tests/python/test_tools_smoke.py

```diff
@@ -424,6 +424,10 @@ def test_vfsstat(self):
     def test_wakeuptime(self):
         self.run_with_duration("wakeuptime.py 1")
 
+    @skipUnless(kernel_version_ge(4,7), "requires kernel >= 4.7")
+    def test_wqlat(self):
+        self.run_with_int("wqlat.py 1 1")
```

Comment:
Please add 'allow_early=True' to run_with_int since the command line may finish earlier.

Reply:
will do that.